README.md

```diff
@@ -7,16 +7,28 @@ Minty is the Unofficial Ruby API for Mint.com.
 
 ## Installation
 
+Currently, `minty` is not entirely stable and is unavailable on Rubygems. You can still clone it down locally and install it via `gem` or `bundler`. You may need to use `bin/minty` instead of just `minty` depending on your installation method and environment.
+
+### Using gem
+
+From the gem's working directory, run:
+
+    $ gem install pkg/minty-*.gem
```

Comment:
Looks good to me, but can you make a note to use the `rake build` command first?

```bash
$ be rake -T build
rake build    # Build minty-0.1.gem into the pkg directory
```